src/connector/src/parser/json_parser.rs

```diff
@@ -76,6 +76,23 @@ impl JsonAccessBuilder {
             json_parse_options,
         })
     }
+
+    pub fn new_for_dynamodb(config: JsonProperties) -> ConnectorResult<Self> {
+        assert!(
+            !config.use_schema_registry,
+            "Dynamodb source does not support schema registry"
+        );
+        let mut json_parse_options = JsonParseOptions::DYNAMODB;
+        if let Some(mode) = config.timestamptz_handling {
+            json_parse_options.timestamptz_handling = mode;
+        }
```

Comment:
I guess the timestamp format is a fixed one, not arbitrary?

Reply:
Currently, this part is not well-designed. 🤔Since we enforce users to use single_blob_column, we don't need to really parse the timestamp data. The only usage for this is when user want to parse timestamp as primary keys or in the future they may want to include system info like `dynamodb.ApproximateCreationDateTime` as additional include columns. I choose to use `GuessNumberUnit` because it accepts more format, including `dynamodb.ApproximateCreationDateTime`'s format.